javascript/src/html_processor.ts

```diff
@@ -20,7 +20,8 @@ import {win} from './win.js';
 
 const assert = console.assert;
 
-const ZWSP = '\u200B'; // U+200B ZERO WIDTH SPACE
+const ZWSP = 0x200b; // U+200B ZERO WIDTH SPACE
+const ZWSP_STR = String.fromCharCode(ZWSP);
```

Comment:
Nit. Possibly something like the following might be closer to what we expect from the constant name?

```typescript
const ZWSP_CODEPOINT = 0x200b;
const ZWSP = String.fromCharCode(ZWSP_CODEPOINT);
```

Reply:
Done.